### PR TITLE
Fixed HA Test Codacy Issues + Tools Codacy issues

### PR DESCRIPTION
**Pull request by ketanarlulkar**
# Problem Statement
- Fixed HA Test Codacy Issues
- Fixed Tools Codacy issues

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [x] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by ketanarlulkar** [comment]
Collection Logs:
```
{'nodeid': 'tests/security/test_cortx_port_scanner_netstat.py::test_cortx_port_scanner_netstat', 'test_id': 'TEST-34218', 'marks': ['security']}] created at C:\Users\528894\Documents\EOS\cortx-test\log\te_meta.json
Successfully unmounted directory

============================================ 2161 tests collected in 7.85s ============================================
(venv)
```

File: tests/ha/test_ha_cluster_health.py [hunk 1]
```diff
@@ -174,7 +174,7 @@ def test_nodes_one_by_one_os_shutdown(self):
         for node in range(self.num_nodes):
             self.restored = False
             node_name = self.srvnode_list[node]
-            LOGGER.info(f"Shutting down {node_name}")
+            LOGGER.info("Shutting down %s", node_name)
```
Comment (ankita-93) [inline tests/ha/test_ha_cluster_health.py:177]: Please add 
pylint: disable=too-many-statements

File: tests/ha/test_ha_cluster_health_gui.py [hunk 2]
```diff
@@ -188,10 +188,9 @@ def test_cluster_one_by_one_os_shutdown_gui(self):
             node_name = self.srvnode_list[node]
             LOGGER.info("Verify if node state online")
             self.ha_gui_obj.verify_node_state(node, "online")
-            LOGGER.info(f"Shutting down {node_name}")
+            LOGGER.info("Shutting down %s", node_name)
```
Comment (ankita-93) [inline tests/ha/test_ha_cluster_health_gui.py:191]: same here. TO disable too-many-statements before test_cluster_one_by_one_os_shutdown_gui

File: tests/ha/test_ha_node_health_gui.py [hunk 3]
```diff
@@ -79,7 +79,7 @@ def setup_class(cls):
 
         # required for Robot_GUI
         cls.ha_gui_obj = HAGUILibs()
-
+        cls.nw_data = None
```
Comment (ankita-93) [inline tests/ha/test_ha_node_health_gui.py:82]: Are not all minor/major warning fixed  for this file ?
I can see some long line warnings are still there in pylint check.

**Review by ankita-93 — COMMENTED** [review]
Few comments for HA file change. Please check once. 
tests/ha/test_ha_node_health_gui.py is still showing minor warning. Please check if possible.

File: tests/ha/test_ha_node_health_gui.py (hunk 3 above)
Comment (ketanarlulkar) [reply tests/ha/test_ha_node_health_gui.py:82]: Where are you seeing those?

**Review by ankita-93 — APPROVED** [review]

**Review by sonalk0209 — APPROVED** [review]

**Review by ravindrachoudhari — APPROVED** [review]

**Pull request merged**